internal/pkg/propagation/summaries.go

```diff
@@ -0,0 +1,761 @@
+	// func TrimFunc(s []byte, f func(r rune) bool) []byte {
+	"bytes.TrimFunc": fromFirstArgToFirstRet,
+	// func TrimPrefix(s, prefix []byte) []byte {
+	"bytes.TrimPrefix": fromFirstArgToFirstRet,
+	// func TrimSuffix(s, suffix []byte) []byte {
+	"bytes.TrimSuffix": fromFirstArgToFirstRet,
+	// func Trim(s []byte, cutset string) []byte {
+	"bytes.Trim": fromFirstArgToFirstRet,
+	// func TrimLeft(s []byte, cutset string) []byte {
+	"bytes.TrimLeft": fromFirstArgToFirstRet,
+	// func TrimRight(s []byte, cutset string) []byte {
+	"bytes.TrimRight": fromFirstArgToFirstRet,
+	// func TrimSpace(s []byte) []byte {
+	"bytes.TrimSpace": fromFirstArgToFirstRet,
+	// func Runes(s []byte) []rune {
+	"bytes.Runes": fromFirstArgToFirstRet,
+	// func Replace(s, old, new []byte, n int) []byte {
+	"bytes.Replace": {
+		ifTainted:   0b101,
+		taintedRets: []int{0},
+	},
+	// func ReplaceAll(s, old, new []byte) []byte {
+	"bytes.ReplaceAll": {
+		ifTainted:   0b101,
+		taintedRets: []int{0},
+	},
+	// func NewReader(b []byte) *Reader {
+	"bytes.NewReader": fromFirstArgToFirstRet,
+	// func WriteString(w Writer, s string) (n int, err error) {
+	"io.WriteString": {
+		ifTainted:   0b10,
+		taintedArgs: []int{0},
+	},
+	// func ReadAtLeast(r Reader, buf []byte, min int) (n int, err error) {
+	"io.ReadAtLeast": {
+		ifTainted:   0b1,
+		taintedArgs: []int{1},
+	},
+	// func ReadFull(r Reader, buf []byte) (n int, err error) {
+	"io.ReadFull": {
+		ifTainted:   0b1,
+		taintedArgs: []int{1},
+	},
+	// func CopyN(dst Writer, src Reader, n int64) (written int64, err error) {
+	"io.CopyN": {
+		ifTainted:   0b10,
+		taintedArgs: []int{0},
+	},
+	// func Copy(dst Writer, src Reader) (written int64, err error) {
+	"io.Copy": {
+		ifTainted:   0b10,
+		taintedArgs: []int{0},
+	},
+	// func CopyBuffer(dst Writer, src Reader, buf []byte) (written int64, err error) {
+	"io.CopyBuffer": {
+		ifTainted:   0b10,
+		taintedArgs: []int{0, 2},
+	},
+	// func LimitReader(r Reader, n int64) Reader {
+	"io.LimitReader": fromFirstArgToFirstRet,
+	// func TeeReader(r Reader, w Writer) Reader {
+	"io.TeeReader": {
+		ifTainted:   0b11,
+		taintedRets: []int{0},
+	},
+	// func MultiReader(readers ...Reader) Reader {
+	"io.MultiReader": fromFirstArgToFirstRet,
+	// func MultiWriter(writers ...Writer) Writer {
+	"io.MultiWriter": fromFirstArgToFirstRet,
+	// func (r *PipeReader) CloseWithError(err error) error {
+	"(*io.PipeReader).CloseWithError": fromFirstArgToFirstRet,
+	// func (w *PipeWriter) CloseWithError(err error) error {
+	"(*io.PipeWriter).CloseWithError": fromFirstArgToFirstRet,
+	// func ReadAll(r io.Reader) ([]byte, error) {
+	"io/ioutil.ReadAll": fromFirstArgToFirstRet,
+	// func NopCloser(r io.Reader) io.ReadCloser {
+	"io/ioutil.NopCloser": fromFirstArgToFirstRet,
+	// func NewReaderSize(rd io.Reader, size int) *Reader {
+	"bufio.NewReaderSize": fromFirstArgToFirstRet,
+	// func NewReader(rd io.Reader) *Reader {
+	"bufio.NewReader": fromFirstArgToFirstRet,
+	// TODO: test
```

Comment:
`[Testing]` This comment highlights this `// TODO`.

Reply:
Oops. Please don't tell anyone how I live.